docs/DeveloperGuide.md

```diff
@@ -1,38 +1,442 @@
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user
+   </td>
+   <td>receive notification any of my previous passwords are reused
+   </td>
+   <td>use a new and different password each time
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user
+   </td>
+   <td>know when I created a certain password
+   </td>
+   <td>keep track of how long my passwords have been used for
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user
+   </td>
+   <td>not have to think of a strong password by myself and let the app handle it
+   </td>
+   <td>know when to change my password as passwords should be changed periodically
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user
+   </td>
+   <td>receive reminder to change passwords that have not been changed for an extended period of time
+   </td>
+   <td>not have to check every single password manually (but still have the ability to) to know when to change my passwords
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user
+   </td>
+   <td>select a particular stored password and expand it to reveal details
+   </td>
+   <td>use the details of the password to make decisions regarding the password (e.g. change the password if it has been used for a long time, create a stronger password the next time if the current one is not strong, etc.)
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user
+   </td>
+   <td>copy and paste my passwords
+   </td>
+   <td>not have to type my password, just simply copy paste
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user
+   </td>
+   <td>search/filter out specific passwords
+   </td>
+   <td>quickly refer to a password that is used for a specific login (e.g. password for NUS email)
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user who wishes to use complex passwords
+   </td>
+   <td>generate complex passwords for me
+   </td>
+   <td>not have to think of a strong password by myself and let the app handle it
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user
+   </td>
+   <td>sync my stored passwords manually
+   </td>
+   <td>have my stored passwords on different devices
+   </td>
+  </tr>
+  <tr>
+   <td>v2.0
+   </td>
+   <td>user
+   </td>
+   <td>encrypt the password.txt file
+   </td>
+   <td>ensure that nobody but me has access to the .txt file that stores my passwords
+   </td>
+  </tr>
+</table>
+
+
+
+---
+
+
+# Non-Functional Requirements
+
+
+
+* Should be responsive and fast, providing quick access to passwords and other information with just one or two commands.
+* Should be user-friendly, with intuitive commands that makes it easy for users to create, store, and retrieve passwords.
+* Should be compatible with various operating systems as long as Java 11 or above is installed.
+* Should be reliable and operational all the time with little-to-no bugs or errors.
+* Should be able to store at least 1000 passwords without compromising performance or security.
```

Comment:
Is the word 'store' missing on the page?
![image](https://user-images.githubusercontent.com/86601208/228770458-65e6050f-518d-4914-b3c3-e5b3cc979c6c.png)